plugin/__init__.py

```diff
@@ -285,6 +285,12 @@ def multi(self, actions):
         return list(map(self.handler, actions))
 
 
+    @util.api()
+    def cardsReviewedToday(self):
```

Comment:
Please rename to `getCardsReviewedToday` to be consistent with *most* APIs that start with a verb.